src/components/MultiShopSelection.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react"
+import {AllItems} from "./ItemSelection"
+import { Items, itemRarities } from "../utilities/types"
+
+const MultiShopSelection: React.FC<{allItems: AllItems, handleItemSelection: (item: Items) => void , numberOfItemsSelected: number}> = ({allItems, handleItemSelection, numberOfItemsSelected}) => {
+
+  const [multiShop, setMultiShop] = useState<Items[]>([])
+
+  const getRandomRarity = () => {
+    const randomNumber: number = Math.floor(Math.random() * 100)
+    if (randomNumber < 70) {
+      return itemRarities.common
+    } else if (randomNumber < 95) {
+      return itemRarities.uncommon
+    } else {
+      return itemRarities.legendary
+    }
+  }
+
+  const getRandomItem = (itemRarityArray: Items[]) => {
+    return Math.floor(Math.random() * itemRarityArray.length)
+  }
+
+  const populateMultiShop = (allItems: AllItems) => {
+    const multiShopRarity = getRandomRarity()
+    const rolledItems = []
+
+    for (let i = 0; i < 3; i++) {
+      const randomItem = getRandomItem(allItems[multiShopRarity].items)
```

Comment:
```suggestion
      const randomItemIndex = getRandomItem(allItems[multiShopRarity].items)
```

Reply:
was incredibly confused why `randomItem` was being used as an index, since i assumed it was an actual item 

(likewise when i saw `randomItem` is the result of `getRandomItem()`, which returns a number